src/main/java/org/usfirst/frc/team2928/Robot.java

```diff
@@ -58,13 +59,13 @@ public void robotInit() {
         autoSelector.addObject("Rotate 90 degrees CCW [Test]", Auto.TEST_ROTATION);
         autoSelector.addObject("Drive 5 feet forward [Test]", Auto.TEST_DISTANCE);
         SmartDashboard.putData("Auto Chooser", autoSelector);
-
         startingPositionSelector = new SendableChooser<>();
         startingPositionSelector.addObject("Left", Field.FieldPosition.LEFT);
         startingPositionSelector.addDefault("Middle", Field.FieldPosition.MIDDLE);
         startingPositionSelector.addObject("Right", Field.FieldPosition.RIGHT);
         SmartDashboard.putData("Starting Position", startingPositionSelector);
-        CameraServer.getInstance().startAutomaticCapture();
+        CameraServer.getInstance().startAutomaticCapture(0);
```

Comment:
Will we have 2 cameras?

Reply:
Yep, we got two cameras now and they all work

Reply:
I'll use intellij git next time, this is my first time really using github